src/github/commit-and-push.ts

```diff
@@ -53,6 +56,15 @@ export function generateTreeObjects(changes: Changes): TreeObject[] {
   return tree;
 }
 
+function* inGroupsOf<T>(
```

Comment:
I like the use of the generator.

nit: since it's only used in one place, I wonder if it's a premature abstraction, might just have gone with `function *commitBatch`.